### PR TITLE
Convert MPI_File_write_ordered.3in - MPI_Free_mem.3in to md

### PR DESCRIPTION
**Pull request by Fangcong-Yin**
Signed-off-by: Fangcong Yin (fyin2@nd.edu)

### CODE REVIEW

**Comment by rhc54** [comment]
ok to test

File: ompi/mpi/man/man3/MPI_Free_mem.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Before the error value is returned, the current MPI error handler is
+called. By default, this error handler aborts the MPI job, except for
+I/O function errors. The error handler may be changed with
+`MPI_Comm_set_errhandler`; the predefined error handler `MPI_ERRORS_RETURN`
+may be used to cause error values to be returned. Note that MPI does not
+guarantee that an MPI program can continue past an error.
+
+# See Also
+
+[`MPI_Alloc_mem``(3)](MPI_Alloc_mem`.html)
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Free_mem.md:64]:
```suggestion
[`MPI_Alloc_mem`(3)](MPI_Alloc_mem.html)
```

Looks like an extra pair of single quotes got in here.

File: ompi/mpi/man/man3/Makefile.am [hunk 2]
```diff
@@ -29,7 +29,14 @@ MD_FILES = \
...
         MPI_Status_f082f.3.md \
-        MPI_Bcast.md
+        MPI_Bcast.md \
+	MPI_File_write_ordered.md \
+	MPI_File_write_ordered_begin.md \
+	MPI_File_write_ordered_end.md \
+	MPI_File_write_shared.md \
+	MPI_Finalize.md \
+	MPI_Finalized.md \
+	MPI_Free_mem.md
```
Comment (jsquyres) [inline ompi/mpi/man/man3/Makefile.am:39]: Please use spaces instead of tabs.

**Review by jsquyres — APPROVED** [review]

**Comment by jsquyres** [comment]
All the content on this PR looks good, and the currently-offline Mellanox CI doesn't specifically test the man pages more than all the other CI's.  So I'm going to go ahead and merge.

**Pull request merged**